docs/debug/index.md

```diff
@@ -47,6 +48,49 @@ iteration    number    sum_so_far    avg_so_far    list_so_far
 6            60        210           35.0          [(10, 10.0), (30, 15.0), (60, 20.0), (100, 25.0), (150, 30.0), (210, 35.0)]
 ```
 
+### While loop Example
+
+To use AccumulationTable with while loops, you need to pass in the name of the loop variable when initializing the Table.
```

Comment:
I left this comment earlier and you missed parts of it:

> Use the code font and class name for AccumulationTable. Again, don't capitalize "While Loops" or "table".

BTW if you aren't sure what "code font" means, I'm referring to using backticks (`) around text to make it appear like code.

Reply:
Thanks David for the clarification!